src/Symfony/Component/Asset/CompilerPass/GitCommitHashCompilerPass.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Asset\CompilerPass;
```

Comment:
this should not be in the Asset component, as there is nothing related to the Asset component in this code (there is code related to the DependencyInjection and Process components)